af_error_reporting/views/assets.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <record id="error_reporting_action" model="ir.actions.act_url">
+            <field name="name">Error reporting</field>
+            <field name="type">ir.actions.act_url</field>
+            <field name="target">self</field>
+            <field name="url">https://serviceportal.arbetsformedlingen.se/sp?id=sc_cat_item&amp;sys_id=733449fedb162300c66a547adc9619cf</field>
+        </record>
+
+        <menuitem action="error_reporting_action"
+                  name="Error reporting"
+                  id="menu_error_reporting"
+                  sequence="999"/>
```

Comment:
Why the high sequence?

Reply:
We want this button to allways be last on this list, as far to the right as possible.